home/PageData.js

```diff
@@ -109,6 +109,11 @@ export default {
         description: 'Drum sequencer with a variety of drum kits and special effects.',
         url: 'shiny-drum-machine/',
       },
+      {
+        title: 'DJ',
```

Comment:
Perhaps the title can be a bit more descriptive? :)

How about "DJ demo" or "Web Audio DJ"?

Reply:
Done